File: Java programming/problems on numbericals/problem7.java
/*
7.Write a program which accept number from user and print even factors of that number. 
		
	Author: Milind Nalawade
	
	Date: 19/06/2021

	language: Java 14.0.2

*/
import java.lang.*;
import java.util.*;
class problem7
{
	public static void main(String args[])
	{
		Scanner sobj=new Scanner(System.in);
		System.out.println("Enter your number : ");
		int no=sobj.nextInt();
		int i=2 ,j=100;
		
		while(j!=0)
		{
			if(i%no==0)
			{
				System.out.print(i+" ");
			}
			i=i+2;
			j--;
		}
	}
}

File: C Programming/problems on bitwise operator/problem7.c
/*

7. Write a program which accept one number from user and off 7th and 10th bit of that number.
	Return modified number. 
	
	Input :  577
	Output : 1 
	
	Author : Milind Nalawade
	
	Date : 02/08/2021
	
	Language : C programming 

*/

#include<stdio.h>
//Logic
int ChkBits(int ino)
{
	int pos=0x00000240;
	int flag=pos&ino;
	if(flag==pos)
	{
		return ino^pos;
	}
	return ino;
}

//Entry point function
int main()
{
	int no=0,ret=0;
	printf("\n Enter the number :");
	scanf("%d",&no);
	ret=ChkBits(no);
	printf("\n Output : %d\n",ret);
	return 0;
}	